packages/modules/src/utils/Types.ts

```diff
@@ -1,44 +1,56 @@
 import { Chain, Hex } from "viem";
-import { UserOperationStruct, SmartAccountSigner } from "@alchemy/aa-core";
+import { SmartAccountSigner, UserOperationStruct } from "@alchemy/aa-core";
 import { SessionKeyManagerModule } from "../SessionKeyManagerModule";
 import { ISessionStorage } from "../interfaces/ISessionStorage.js";
 import { SupportedSigner } from "@biconomy/common";
 export type ModuleVersion = "V1_0_0"; // | 'V1_0_1'
 
 export interface BaseValidationModuleConfig {
+  /** entryPointAddress: address of the entry point */
   entryPointAddress?: Hex;
 }
 
 export interface ECDSAOwnershipValidationModuleConfig extends BaseValidationModuleConfig {
+  /** Address of the module */
   moduleAddress?: Hex;
+  /** Version of the module */
   version?: ModuleVersion;
+  /** Signer: viemWallet or ethers signer. Ingested when passed into smartAccount */
   signer: SupportedSigner;
 }
 
 export interface ECDSAOwnershipValidationModuleConfigConstructorProps extends BaseValidationModuleConfig {
+  /** Address of the module */
   moduleAddress?: Hex;
+  /** Version of the module */
   version?: ModuleVersion;
+  /** Signer: Converted from viemWallet or ethers signer to SmartAccountSigner */
   signer: SmartAccountSigner;
 }
 
 export interface SessionKeyManagerModuleConfig extends BaseValidationModuleConfig {
+  /** Address of the module */
   moduleAddress?: Hex;
+  /** Version of the module */
   version?: ModuleVersion;
-  nodeClientUrl?: string;
```

Comment:
only idea to keep this was to have a client which can call managed backend + db where sessions are being stored. But I guess custom implementation of ISeesionStorageClient can implement and handle it's own dto and api calls

Reply:
as above